keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -199,6 +201,14 @@ tolerations:
       effect: "NoSchedule"
 ```
 
+#### Memory Overprovisioning
+
+Windows always treats all user-mode memory allocations as virtual, and pagefiles are mandatory. The net effect is that Windows won't reach out of memory conditions the same way Linux does, and processes will page to disk instead of being subject to out of memory (OOM) termination. There is no way to guarantee a physical memory allocation or reserve for a process. See [#73417](https://github.com/kubernetes/kubernetes/issues/73417) for more details on the investigation for 1.14.
+
+Keeping memory usage within reasonable bounds is possible using a combination of kubelet parameters `--kubelet-reserve` / `--system-reserve`, and resource limits on containers. These will be documented as best practices for v1.14. The related kubelet parameters `--eviction-hard`, `--eviction-soft`, and `--enforce-node-allocatable` are invalid for v1.14.
```

Comment:
Few comments:

- `--kubelet-reserve` / `--system-reserve` does affect [node allocatable](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable) but does not put constriants on system or kubelete resources.
-  `--eviction-hard` does affect node allocatable as well but again has no affect on eviction.